modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureListView.testcase

```diff
@@ -34,188 +38,172 @@ definition {
+		}
 
-		WaitForElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECENT_CREATE_DATE");
+		task ("Then the date the digital signature was created is shown") {
+			Refresh();
 
-		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_RECENT_CREATE_DATE");
+			DigitalSignature.viewCreateDate();
 
-		DigitalSignature.deleteEnvelope(envelopeName = "Poshi Test CanViewCorrectDate");
+			DigitalSignature.deleteEnvelope(envelopeName = "Poshi Test CanViewCorrectDate");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		DMDocument.deleteCP(dmDocumentTitle = "DM Document Title");
+			DMDocument.deleteCP(dmDocumentTitle = "DM Document Title");
 
-		RecycleBin.openRecycleBinAdmin(siteURLKey = "guest");
+			RecycleBin.openRecycleBinAdmin(siteURLKey = "guest");
 
-		RecycleBin.emptyCP();
+			RecycleBin.emptyCP();
+		}
 	}
 
-	@description = "This is a test for LPS-131206. Verify if its possible view envelope details"
+	@description = "LPS-131206. Verify if its possible view envelope details"
 	@priority = 4
 	test CanViewEnvelopeDetails {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		Navigator.openURL();
-
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Digital Signature");
+		task ("When a User Test Goes to Digital Signature") {
+			Navigator.openURL();
 
-		var envelopeName = "Test";
-
-		Click(
-			envelopeName = ${envelopeName},
-			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ENVELOPE_NAME");
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Digital Signature");
+		}
 
-		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ENVELOPE_ID");
+		task ("Then you can see details of the envelope") {
+			DigitalSignature.viewEnvelopeDetails(documentName = "Test");
+		}
 	}
 
-	@description = "This is a test for LPS-131206. Verify if Digital Signature can be enabled"
+	@description = "LPS-131206. Verify if Digital Signature can be enabled"
 	@priority = 5
 	test DigitalSignatureCanBeEnabled {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
+		task ("When a User Test Goes to Digital Signature") {
+			Navigator.openURL();
 
-		Navigator.openURL();
-
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Digital Signature");
-
-		var dsStatus = "Sent";
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Digital Signature");
+		}
 
-		AssertElementPresent(
-			dsStatus = ${dsStatus},
-			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_STATUS");
+		task ("Then the status is checked") {
+			DigitalSignature.viewDigitalSignaturestatus(viewStatus = "Sent");
+		}
 	}
 
-	@description = "This is a test for LPS-137169. Verify if have more recipients than one"
+	@description = "LPS-131206. Verify if have more recipients than one"
```

Comment:
Here you can change the test description to: `Verify if is possible view on list when an envelope have more recipients than one`